sys/linux/bpf_386.const

```diff
@@ -153,7 +153,7 @@ BPF_XOR0 = 10
 BTF_MAGIC = 60319
 BTF_VERSION = 1
 MAX_BPF_REG = 11
-__BPF_FUNC_MAX_ID = 98
+__BPF_FUNC_MAX_ID = 93
```

Comment:
You seem to generate this on some old tree. Const files need to be generated on the latest Linus tree HEAD. __BPF_FUNC_MAX_ID must not go down. Please regenerate.